modules/compiler/builtins/cmake/FindBuiltinsTools.cmake

```diff
@@ -60,7 +67,7 @@ function(find_builtins_tools tools_dir)
       )
       set(BUILTINS_LLVM_CMAKE ${BUILTINS_LLVM_CMAKE}/LLVMConfig.cmake)
     endif()
-    if(NOT EXISTS BUILTINS_LLVM_CMAKE)
+    if(NOT EXISTS ${BUILTINS_LLVM_CMAKE})
```

Comment:
This change looks suspicious. The `if` command is special and takes variable names, and will implicitly check their values. Why is `${`...`}` needed?

Reply:
I was confused by this as well, without `${...}` I get the warning `builtins tools installation does not...`, I tried printing out the content of `BUILTINS_LLVM_CMAKE` on my side, and it is a valid absolute path. Adding the `${...}` removed the warning, but this change is not strictly necessary for this PR, so I've fine with reverting it and looking more into this.

Reply:
Ignore my comment, this change is correct, please keep it in. The special rule for `if` does not apply to `if([NOT] EXISTS)`.